### ISSUE TITLE
gradle lint shows 'Package not included in Android' for jjwt-0.9.0.jar

### ISSUE THREAD

**Issue by MikeMitterer** [jwtk/jjwt#312]
```
.gradle/caches/modules-2/files-2.1/io.jsonwebtoken/jjwt/0.9.0/64e3ed8a5f0bc93ecea592a0b3280f995efbdfe9/jjwt-0.9.0.jar: Invalid package reference in library; not included in Android: javax.xml.bind. Referenced from io.jsonwebtoken.impl.Base64Codec.
This check scans through libraries looking for calls to APIs that are not included in Android.
```
Any ideas why this happens?

**Comment by ycoupe** [comment]
Same for me, did you find a trick ? @MikeMitterer

**Comment by hashlin** [comment]
+1 Same here

**Comment by lhazlewood** [comment]
@MikeMitterer @ycoupe @hashlin JJWT .jars are used in both normal Java projects as well as Android projects.  As a result, JJWT will automatically depend on JAXB APIs if not running in an Android runtime environment.  Here's the logic:

https://github.com/jwtk/jjwt/blob/master/src/main/java/io/jsonwebtoken/impl/DefaultTextCodecFactory.java#L45-L49

As you can see, even though JJWT use non-Android-compatible classes, it only does so when not running in an Android application.

Based on this, is there any work for the JJWT to do?  Or can this issue be closed?

**Comment by lhazlewood** [comment]
I wonder if this will be cleaned up with #333

**Comment by lhazlewood** [comment]
This is now resolved via #333 and in master.  A new release with this change will happen asap, hopefully in the next few days, but most definitely before the end of the month.

**Comment by lhazlewood** [comment]
This has been fixed and released in 0.10.0.
